lib/absinthe/adapter/language_conventions.ex

```diff
@@ -73,11 +75,15 @@ defmodule Absinthe.Adapter.LanguageConventions do
     "__" <> to_external_name(underscored_name, role)
   end
 
+  def to_external_name(underscored_name, :operation) when is_binary(underscored_name) do
```

Comment:
Does this imply a change in behavior? Seems like previously an operation name would be transformed?

I imagine this isn't a breaking change in the usual sense, but I suspect some APIs might track the operation name for monitoring, etc and could be relying on the existing behavior - say you have a dashboard tracking API usage based on operation name...

Reply:
It's been several months since I made these changes, but I'll try to remember...

I think the previous version was unnecessarily performing `Utils.camelize(name, lower: true)` on things like `query`, `mutation`, and `subscription`.

This change was made to match the `to_internal_name` behavior above where the `:operation` names are left untouched.  With this change in place `to_internal_name` and `to_external_name` are roughly roundtrip versions of each other, which is how the strict version of the adapter operates (by checking that they do in fact match).  Only those using custom operations would be affected and I'm not even sure whether this was even working correctly in the first place.

Reply:
So I looked around and couldn't find a use of `to_external_name` or `to_internal_name` using `:operation`... Also looked in `absinthe_plug`

I even put a `raise` in both functions and all the tests pass just fine.

I suspect we could just remove `:operation` from the `@type role_t` and remove the function heads. I looked back and `to_internal_name` used to be called in the inner workings of `Adapter` but that doesn't happen anymore

cc @bruce

Reply:
I think that's fine.

Reply:
I'll follow up with a quick PR to remove those